app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(commands_str: str) -> None:
+    commands_list = commands_str.split()
+    file_out = commands_list[1]
+    file_in = commands_list[2]
```

Comment:
In this way looks better
```suggestion
    commands_list, file_out, file_in = commands_str.split()
```